.github/workflows/cspell.yml

```diff
@@ -0,0 +1,24 @@
+name: CSpell
+on: push
+jobs:
+  cspell:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v2-beta
```

Comment:
±: Would you explain the pin to `@v2-beta`? It's a few years out of date, and I strongly recommend updating to v2 or v3 if possible.

Reply:
only for the reference of the previous actions, but it is a good suggestion, I changed it